src/contacts.cs

```diff
@@ -441,6 +452,795 @@ interface CNLabelContactRelationKey {
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationAuntMothersYoungerSister")]
+		NSString AuntMothersYoungerSister { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationAuntMothersElderSister")]
+		NSString AuntMothersElderSister { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationAuntMothersBrothersWife")]
+		NSString AuntMothersBrothersWife { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationGrandaunt")]
+		NSString Grandaunt { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncle")]
+		NSString Uncle { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleParentsBrother")]
+		NSString UncleParentsBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleParentsYoungerBrother")]
+		NSString UncleParentsYoungerBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleParentsElderBrother")]
+		NSString UncleParentsElderBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleMothersBrother")]
+		NSString UncleMothersBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleMothersYoungerBrother")]
+		NSString UncleMothersYoungerBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleMothersElderBrother")]
+		NSString UncleMothersElderBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleMothersSistersHusband")]
+		NSString UncleMothersSistersHusband { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleFathersBrother")]
+		NSString UncleFathersBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleFathersYoungerBrother")]
+		NSString UncleFathersYoungerBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleFathersElderBrother")]
+		NSString UncleFathersElderBrother { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleFathersSistersHusband")]
+		NSString UncleFathersSistersHusband { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleFathersYoungerSistersHusband")]
+		NSString UncleFathersYoungerSistersHusband { get; }
+
+		[Watch (6,0), Mac (10,15, onlyOn64: true), iOS (13,0)]
+		[Field ("CNLabelContactRelationUncleFathersElderSistersHusband")]
+		NSString UncleFathersElderSistersHusband { get; }
```

Comment:
These relations are insane! Everything looks good to me though! Perhaps remove the newline before the curly braces when starting interfaces though for style, but your call!

Reply:
Updated to remove extra lines :)